package.json

```diff
@@ -36,7 +36,11 @@
   },
   "license": "MIT",
   "scripts": {
-    "test": "mocha -R spec --check-leaks",
+    "test": "mocha -R spec --check-leaks --globals __core-js_shared__",
     "coverage": "istanbul cover ./node_modules/mocha/bin/_mocha"
+  },
+  "dependencies": {
+    "babel-core": "~6.26.0",
```

Comment:
Any reason why you're not using the `^` operator? Personally, I prefer that.

Reply:
I personally prefer to use fixed or `~`. A package should not contain breaking changes on minor updates but it often does. I find it more secure to make it fixed or set it to patch updates. Let me know if I still need to change is :).